github/scim.go

```diff
@@ -17,6 +17,25 @@ import (
 // GitHub API docs: https://docs.github.com/rest/scim
 type SCIMService service
 
+// SCIMGroupAttributes represents supported SCIM Group attributes.
+//
+// GitHub API docs: https://docs.github.com/en/enterprise-cloud@latest/rest/enterprise-admin/scim#list-provisioned-scim-groups-for-an-enterprise
+type SCIMGroupAttributes struct {
+	DisplayName *string                 `json:"displayName,omitempty"` // The name of the group, suitable for display to end-users. (Optional.)
+	Members     []*SCIMDisplayReference `json:"members,omitempty"`     // (Optional.)
+	Schemas     []string                `json:"schemas,omitempty"`     // (Optional.)
+	ExternalID  *string                 `json:"externalId,omitempty"`  // (Optional.)
+	// Only populated as a result of calling ListSCIMProvisionedIdentitiesOptions:
+	ID   *string   `json:"id,omitempty"`
+	Meta *SCIMMeta `json:"meta,omitempty"`
+}
+
+type SCIMDisplayReference struct {
```

Comment:
Please add a godoc-style comment to this struct with complete punctuation for the docs website.  Thanks.

Reply:
Added doc string for struct